human_experiments/scripts/stop_data_streaming_processes

```diff
@@ -71,13 +71,27 @@ __stop_on_client() {
   fi
 }
 
+__stop_monitoring_program_cat() {
+  if [[ $ALL -ne 1 ]]; then
+    yes_no_question "${padding} Do you want to stop image timestamp monitoring?"
+    ans=$?
+    padding="$padding "
+  fi
+  if [[ $ALL -eq 1 ]] || [[ $ans -eq 0 ]]; then
+    ssh cat@cat "~/Desktop/image_timestamp_scripts/end_image_timestamp.sh"
```

Comment:
This should be under `~git` as I mentioned in another item of this review. Shouldn't `end_image_timestamp.sh` be under `tomcat/human_experiments/scripts`?

Reply:
This will be done in the next pull request.